### ISSUE TITLE
Feature Enhancement: Include the list of computers returned by query before confirm

### ISSUE THREAD

**Issue by han44-illinois** [engrit-illinois/Invoke-TaskSequence#1]
This module has the helpful confirmation feature to list details regarding the deployment ID before asking the user to confirm, but I noticed that it does not include details about what computers are returned from what is fed into `-ComputerNames`

This might be relatively obvious, but since the examples show use cases that encourage using variables to feed into this parameter (and I usually do it this way anyway) it would be useful to also essentially show the contents of what's fed into the parameter when asking for confirmation.

Low priority.

**Comment by han44-illinois** [comment]
Hey actually, I realized this information does appear. I must have started reading only after the giant mishmash of computer names. Closing this issue
